src/pages/registration/references_tab/components/SearchContainerField.tsx

```diff
@@ -34,70 +35,109 @@ const SearchContainerField = (props: SearchContainerFieldProps) => {
-              placeholder={props.placeholder}
-              dataTestId="container-search-field"
-              showSearchIcon
-              errorMessage={meta.touched && !!meta.error ? meta.error : undefined}
-            />
-          )}
-        />
+    <>
+      <Field name={props.fieldName}>
+        {({ field, meta }: FieldProps<string>) => (
+          <Autocomplete
+            {...autocompleteTranslationProps}
+            popupIcon={null}
+            options={options}
+            onBlur={() => setFieldTouched(field.name)}
+            onInputChange={(_, newInputValue) => setSearchTerm(newInputValue)}
+            getOptionSelected={(option, value) => option.id === value.id}
+            value={selectedContainer}
+            onChange={(_, inputValue) => {
+              if (inputValue) {
+                // Construct IRI manually, until it is part of the object itself
+                setFieldValue(field.name, `${API_URL}${getRegistrationPath(inputValue.id)}`);
+              } else {
+                setSearchTerm('');
+                setFieldValue(field.name, '');
+              }
+            }}
+            loading={isLoadingSearchContainerOptions || isLoadingSelectedContainer}
+            getOptionLabel={(option) => option.title}
+            renderOption={(option, state) => (
+              <StyledFlexColumn>
+                <Typography variant="subtitle1">
+                  <EmphasizeSubstring text={option.title} emphasized={state.inputValue} />
+                </Typography>
+                <Typography variant="body2" color="textSecondary">
+                  <Truncate lines={1}>
+                    {option.publicationDate.year && displayDate(option.publicationDate)}
+                    {option.publicationDate.year && option.contributors.length > 0 && ' - '}
+                    {option.contributors.map((contributor) => contributor.name).join('; ')}
+                  </Truncate>
+                </Typography>
+              </StyledFlexColumn>
+            )}
+            renderInput={(params) => (
+              <AutocompleteTextField
+                {...params}
+                label={props.label}
+                required
+                isLoading={isLoadingSearchContainerOptions || isLoadingSelectedContainer}
+                placeholder={props.placeholder}
+                dataTestId="container-search-field"
+                showSearchIcon
+                errorMessage={meta.touched && !!meta.error ? meta.error : undefined}
+              />
+            )}
+          />
+        )}
+      </Field>
+      {selectedContainer?.reference?.publicationContext && (
+        <SelectedContainerSummary publicationContext={selectedContainer.reference.publicationContext} />
       )}
-    </Field>
+    </>
+  );
+};
+
+interface SelectedContainerSummaryProps {
+  publicationContext: SearchPublicationContext;
+}
+
+const SelectedContainerSummary = ({ publicationContext }: SelectedContainerSummaryProps) => {
```

Comment:
denne komponenten er (forhåpentligvis) veldig midlertidig. Kine må finne ut hvordan vi skal vise dette, så må vi nok lage en felles måte å vise det for journal, publisher, linkedContext, og corrigendumFor etterpå